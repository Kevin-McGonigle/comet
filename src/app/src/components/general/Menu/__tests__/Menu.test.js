import React from 'react';
import ToolbarMenu from '../Menu';
import { shallow } from 'enzyme';
import "../../../../setupTests"

test('should render ToolbarMenu succesfully', () => { 
    const component = shallow(<ToolbarMenu/>);
    expect(component.html()).toMatchSnapshot();
});

