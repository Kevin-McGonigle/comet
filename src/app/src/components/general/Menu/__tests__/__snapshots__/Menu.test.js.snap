// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ToolbarMenu succesfully 1`] = `"<button class=\\"css-69cngj ðŸ“¦fnt-fam_b77syt ðŸ“¦mt_0px ðŸ“¦fnt-sze_12px ðŸ“¦f-wght_500 ðŸ“¦ln-ht_32px ðŸ“¦ltr-spc_0 ðŸ“¦btrr_3px ðŸ“¦bbrr_3px ðŸ“¦btlr_3px ðŸ“¦bblr_3px ðŸ“¦pt_0px ðŸ“¦pb_0px ðŸ“¦pr_16px ðŸ“¦pl_16px ðŸ“¦ml_0px ðŸ“¦mr_16px ðŸ“¦mb_0px ðŸ“¦h_32px ðŸ“¦pst_relative ðŸ“¦dspl_inline-flex ðŸ“¦algn-itms_center ðŸ“¦flx-wrap_nowrap ðŸ“¦box-szg_border-box\\" role=\\"button\\" aria-haspopup=\\"true\\">Menu</button>"`;
